src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -744,25 +743,14 @@ std::shared_ptr<ov::ICompiledModel> Plugin::import_model(std::istream& stream, c
         }
         _logger.debug("Successfully read %zu bytes into blob.", graphSize);
 
-        auto meta = compiler->parse(blob, localConfig);
-        // If graphHandle is not a nullptr it means there is still an instance of the blob maintained inside the driver
-        // and we can release the copy of the blob here to reduce memory consumption.
-        if (meta.graphHandle != nullptr) {
-            blob.clear();
-            blob.shrink_to_fit();
-        }
+        auto graph = compiler->parse(blob, localConfig);
```

Comment:
The blob is prepared in this function, but with IGraph, we can not release the blob easily here. We do not know if the IGraph has real blob content inside(CiPGraph contains blob, but CiDGraph only use graphHandle), This is one regression of IGraph,  we can not simply move() to save memory